saleor/payment/utils.py

```diff
@@ -977,6 +985,10 @@ def parse_transaction_action_data(
     )
 
 
+def truncate_message(message: str):
+    return message[:509] + "..." if len(message) >= 512 else message
```

Comment:
shouldn't be `if len(message) > 512` or `if len(message) >= 513`?  Right now It looks like that providing message with 512 characters will cut it to 509 🤔